test/e2e/common_test.go

```diff
@@ -236,8 +239,15 @@ func PodmanTestCreateUtil(tempDir string, remote bool) *PodmanTestIntegration {
 	}
 	os.Setenv("DISABLE_HC_SYSTEMD", "true")
 	CNIConfigDir := "/etc/cni/net.d"
+	netavarkConfigDir := "/etc/containers/networks"
+	if _, err := os.Stat(root); os.IsNotExist(err) {
+		if err := os.MkdirAll(root, 0755); err != nil {
+			panic(err)
+		}
+	}
```

Comment:
Why do we even force the default here, could we just not set it at all.

Reply:
I never understood why the original was there so I added it.  I think because it is being used in the overall command line ?